### ISSUE TITLE
Homevent VOC / air quality control 

### ISSUE THREAD

**Issue by Zomkidom** [zittix/Hoval-GW#1]
Hi,

Is there a way to turn on/off the VOC functionality? Aka “air quality control”

Thank you!

**Comment by zittix** [comment]
Yes it should be possible but I don't have this device so I don't know its address. If you can provide a candump of your installation I can have a look.
